lesson_01/TommyTran/README.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+
+<link href="style.css" rel="stylesheet"/>
+
+
+<body>
+ <main>
+    <div class="md-heading">
+ <h1> Tommy Tran README</h1>
+    </div>
+    <div class="line"> </div>
+    <div class="md-heading">
+<h2>About Me</h2>
```

Comment:
A vertical line should appear beneath your headings, here and below (just like in the markdown version on GitHub).